### PR TITLE
C libs classifier(APP-120)

### PR DESCRIPTION
**Pull request by yaronskaya**

### CODE REVIEW

File: src/main/kotlin/app/extractors/ExtractorInterface.kt [hunk 1]
```diff
@@ -278,6 +278,10 @@ interface ExtractorInterface {
             return emptyList()
         }
 
+        if (languageLabel == "c" && languageLabel in selectedCategories) {
```
Comment (anatolykislov) [inline src/main/kotlin/app/extractors/ExtractorInterface.kt]: What this block do?

Comment (yaronskaya) [reply src/main/kotlin/app/extractors/ExtractorInterface.kt]: It is specific for C language.

Comment (anatolykislov) [reply src/main/kotlin/app/extractors/ExtractorInterface.kt]: I see. But what it do? No lines libraries for c language?

Comment (yaronskaya) [reply src/main/kotlin/app/extractors/ExtractorInterface.kt]: No, just for a line that has language label selected.

Comment (anatolykislov) [reply src/main/kotlin/app/extractors/ExtractorInterface.kt]: Constant already defined: `if (languageLabel == CExtractor.LANGUAGE_NAME && ... )`
Also, add comments about percentile, please.

**Pull request merged**